api-server/src/events/events.authenticated.controller.ts

```diff
@@ -96,10 +103,19 @@ export default class EventsAuthenticatedController {
     type: String,
     description: 'Filter events by category',
   })
+  @ApiQuery({
+    name: 'owning_partner_id',
+    description:
+      'Filter events by owning partner ID. Can be specified multiple times.',
```

Comment:
Nitpick: would this be clearer as "More than one partner ID can be specified" or "Supports array syntax to return events for more than one partner at a time"?